docs/snp.md

```diff
@@ -73,6 +77,10 @@ the following command:
 ```
 ./snp.sh launch-guest
 ```
+On the Coconut-SVSM setup, use the following command:
+```
+./snp.sh --svsm launch-guest
+```
```

Comment:
```suggestion
The `--svsm` option can be specified in the above command to launch a Coconut-SVSM guest. The `setup-host` command must be run with this same option in order to have the necessary dependencies to launch an SVSM guest.
```

Reply:
You are asking remove command and add verbal explanation. I would like to keep command as well in documentation. This gives clear instruction on where they should add "--svsm" option. @ryansavino, any suggestions?